ttnn/cpp/ttnn/operations/reduction/accumulation/ema/kernels/dataflow/ema_reader.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```
stdint.h is a C version of the file. Ideally, we should use C++ one.